internal/pkg/manifest/lb_web_svc.go

```diff
@@ -162,6 +162,11 @@ func (s *LoadBalancedWebService) BuildArgs(wsRoot string) *DockerBuildArgs {
 	return s.ImageConfig.Image.BuildConfig(wsRoot)
 }
 
+// EnvFiles returns the env file name in a ws service directory.
+func (s *LoadBalancedWebService) EnvFiles() string {
```

Comment:
```suggestion
// EnvFile returns the path in the workspace of the environment file associated with the service.
func (s *LoadBalancedWebService) EnvFile() string {
```

Reply:
this func is not supposed to be included in the PR sorry...